extconf.rb

```diff
@@ -132,11 +132,11 @@ def rule(target, clean = nil)
 else
    version = 0
 end
-if version < 73
+if version < 92
    raise <<-EOT
 
 ============================================================
-#{version_str} is unsupported.  Try plruby-0.4.2.
+#{version_str} is not supported anymore.  Try plruby-0.5.7 or older.
```

Comment:
I think it is rather ugly that this overflows outside the equals signs. I suggest just dropping the "anymore" because it is obvious from the context.

Reply:
Agreed.